### ISSUE TITLE
Char single-quote and double-quote escape 

### ISSUE THREAD

**Issue by ZenVoich** [dfinity/motoko#3984]
`Debug.print(debug_show('\"'));` => `"` - good?

`Debug.print(debug_show('"'));` => syntax error [M0002], malformed operator, should be allowed?

`Debug.print(debug_show('''));` => `'` - should be syntax error?

`Debug.print(debug_show('\''));` => `'` - good

**Comment by crusso** [comment]
Yes, this looks like it needs reconsidering. Thanks for reporting!

**Comment by crusso** [comment]
Actually, on second thought. I think this behaviour is actually fine since we know that a Char literal must contain exactly one Character.
